packages/ui/src/components/ValidatorList/KeystoresDataGrid.tsx

```diff
@@ -61,6 +69,106 @@ export default function KeystoresDataGrid({
     openDashboardTab();
   }, [validatorSummaryURL]);
 
+  useState<CustomValidatorUpdateRequest>();
+  const [validatorsSubscriptionMap, setValidatorsData] =
+    useState<SmoothStatusByPubkey>({});
```

Comment:
I would consider as "dangerous" setting a initial value an empty object, it might lead to future issues since it will be considered as a non-nullish value

Reply:
you're right. changed it to be initialized as `null`, not in danger of being considered non-nullish anymore.